subprojects/language-java/src/main/java/org/gradle/api/tasks/JavaExec.java

```diff
@@ -727,6 +743,18 @@ public List<CommandLineArgumentProvider> getJvmArgumentProviders() {
         return javaExecSpec.getJvmArgumentProviders();
     }
 
+    /**
+     * Returns the list of configured extra JVM arguments.
+     *
+     * @return A property of extra JVM arguments.
+     * @since 8.1
+     */
+    @Input
+    @Incubating
+    public ListProperty<String> getJvmArguments() {
```

Comment:
We also need to add this to our DSL docs, see `src/docs/dsl/org.gradle.api.tasks.JavaExec.xml`